packages/api/src/dwn-api.ts

```diff
@@ -2,10 +2,12 @@ import type { DwnResponse, Web5Agent } from '@web5/agent';
 import type {
   RecordsReadOptions,
   RecordsQueryOptions,
+  RecordsQueryReply,
   RecordsWriteMessage,
   RecordsWriteOptions,
   RecordsDeleteOptions,
   ProtocolsQueryOptions,
+  ProtocolsQueryReply,
   RecordsQueryReplyEntry,
   ProtocolsConfigureMessage,
   ProtocolsConfigureOptions,
```

Comment:
Super minor nit but imports are sorted by length in the monorepo:
```suggestion
  RecordsQueryReply,
  RecordsReadOptions,
  ProtocolsQueryReply,
  RecordsQueryOptions,
  RecordsWriteMessage,
  RecordsWriteOptions,
  RecordsDeleteOptions,
  ProtocolsQueryOptions,
  RecordsQueryReplyEntry,
  ProtocolsConfigureMessage,
  ProtocolsConfigureOptions,
```